package.json

```diff
@@ -25,20 +27,31 @@
     "watch": "npm run build -- --watch",
     "watch:test": "npm run test -- --watch"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@types/mongodb": "^3.1.19",
+    "mongodb": "^3.1.10",
+    "tyranid": "^0.5.66"
+  },
```

Comment:
We should probably convert these to [`peerDependencies`](https://nodejs.org/en/blog/npm/peer-dependencies/) in the future so that consuming apps don't run into version mismatches and/or multiple imported copies of these libs.